README.md

```diff
@@ -65,38 +65,52 @@ NOTE: OpenCL host side program should be linked with icd loader `-lOpenCL`.
 ```
 $ <pocl-install-dir>/bin/poclcc -l
 
+// The following output should be shown:
 LIST OF DEVICES:
 0:
   Vendor:   THU
     Name:   Ventus GPGPU device
  Version:   2.2 HSTR: THU-ventus-gpgpu
 ```
 
-Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example(Work in progress).
+Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example:
+```
+$ ./<pocl-install-dir>/examples/example1
+
+// The following output should be shown:
+(0.000000, 0.000000, 0.000000, 0.000000) . (0.000000, 0.000000, 0.000000, 0.000000) = 0.000000
+(1.000000, 1.000000, 1.000000, 1.000000) . (1.000000, 1.000000, 1.000000, 1.000000) = 4.000000
+(2.000000, 2.000000, 2.000000, 2.000000) . (2.000000, 2.000000, 2.000000, 2.000000) = 16.000000
+(3.000000, 3.000000, 3.000000, 3.000000) . (3.000000, 3.000000, 3.000000, 3.000000) = 36.000000
+OK
+```
 
 
 ### 4: Compiler using example
 
-we can now use our built compiler to generate an ELF file, and using [spike](https://github.com/THU-DSP-LAB/ventus-gpgpu-isa-simulator) to complete the isa simulation.
+We can now use our built compiler to generate an ELF file, and using [spike](https://github.com/THU-DSP-LAB/ventus-gpgpu-isa-simulator) to complete the isa simulation.
 
 > Cause the address space requirement in spike, we use a customized linker script for our compiler
 
-Take `vecadd.cl` below as an example :
```

Comment:
"Take the following `vecadd.cl` as an example and place it under `<path-to-llvm-ventus>`."
Maybe this is more appropriate, and then just change line 99?

Reply:
Sure, now I rephrase it as "First, name the following program `vecadd.cl`, and place it under `<path-to-llvm-ventus>`".